flickr_download/flick_download.py

```diff
@@ -255,7 +267,7 @@ def print_sets(username):
     @param username: str,
     """
     with Timer('findByUserName()'):
```

Comment:
```suggestion
    with Timer('find_user()'):
```

Reply:
(hmm, not sure how there was a spacing change too... that was not intended :)

Reply:
Sure, I can change that. Should I also add Timer also to other calls to find_user?

Reply:
No, it's good at is. But thanks for asking!